roles/custom/matrix-bot-draupnir/defaults/main.yml

```
@@ -75,6 +75,13 @@ matrix_bot_draupnir_raw_homeserver_url: ""
 # Its Exposed here because its common enough to be valid to expose.
 matrix_bot_draupnir_disable_server_acl: "false"
 
+# Used to control if the Synapse Admin API is exposed internally to the containers and therefore giving Draupnir Access.
+matrix_bot_draupnir_admin_api_enabled: ""
```

```suggestion
matrix_bot_draupnir_admin_api_enabled: false
```

It's odd that this appears to be boolean, but its default value in the role is an empty string.